.github/workflows/main.yml

```diff
@@ -31,19 +36,18 @@ jobs:
         with:
           submodules: recursive
       
+      - uses: risc0/risc0/.github/actions/sccache@release-0.21
+        with:
+          key: Linux-default
+
       - name: Install rust
-        uses: risc0/risc0/.github/actions/rustup@release-0.20
+        uses: risc0/risc0/.github/actions/rustup@release-0.21
 
       - name: Install Foundry
         uses: risc0/foundry-toolchain@2fe7e70b520f62368a0e3c464f997df07ede420f
 
-      - name: Install cargo-install
-        uses: risc0/cargo-install@v1
-        with:
-          crate: cargo-binstall
-
-      - name: Cargo binstall cargo-risczero
-        run: cargo binstall -y --force cargo-risczero@${{ env.RISC0_CRATE_VERSION }}
+      - name: Cargo install cargo-risczero
+        run: cargo install --force --locked cargo-risczero@$RISC0_CRATE_VERSION
```

Comment:
IIUC, this will take quite a bit longer than installing and using `binstall` because it'll require building the prover, which is quite slow at the moment. Is there a particular reason to do it this way instead?